### ISSUE TITLE
Build fails in Ubuntu 18.04

### ISSUE THREAD

**Issue by tavasti** [olegkapitonov/Kapitonov-Plugins-Pack#11]
Can't build latest git tree in ubuntu 18.04

tavasti@hermo:~/git/Kapitonov-Plugins-Pack$ meson build
The Meson build system
Version: 0.45.1
Source dir: /home/tavasti/git/Kapitonov-Plugins-Pack
Build dir: /home/tavasti/git/Kapitonov-Plugins-Pack/build
Build type: native build

meson.build:24:9: ERROR: lexer
plugins = {
         ^

A full log can be found at /home/tavasti/git/Kapitonov-Plugins-Pack/build/meson-logs/meson-log.txt

**Comment by trebmuh** [comment]
A shot in the dark: try with a newer meson version.

**Comment by olegkapitonov** [comment]
@tavasti, thank you for the information!

Yes, @trebmuh is right, I use meson 0.53 to build KPP in Ubuntu 18.04 (this binary version https://github.com/olegkapitonov/Kapitonov-Plugins-Pack/releases/download/v1.1/KPP-1.1-binary-ubuntu-bionic.tar.bz2)

meson 0.45 which is in the repository can't build current version of KPP.

You may try to build meson 0.53 from the source first (as I do).

**Comment by tavasti** [comment]
Ok, pip3 install meson fixes that part. 

Next problem:
```
Program faust found: YES (/usr/bin/faust)

meson.build:15:0: ERROR: Command "/usr/bin/faust -archdir" failed with status 255.

A full log can be found at /home/tavasti/git/Kapitonov-Plugins-Pack/build/meson-logs/meson-log.txt
tavasti@hermo:~/git/Kapitonov-Plugins-Pack$ /usr/bin/faust -archdir
ERROR : unrecognized or incorrectly defined option "-archdir"
tavasti@hermo:~/git/Kapitonov-Plugins-Pack$ faust --version
FAUST, DSP to C++ compiler, Version 0.9.95
Copyright (C) 2002-2016, GRAME - Centre National de Creation Musicale. All rights reserved. 
```

**Comment by trebmuh** [comment]
You probably need a >2 version of faust. 0.9.95 sounds to be something like 4 years old now.

**Comment by tavasti** [comment]
That is faust coming from Ubuntu 18.04 repos. So most likely that is version which was considered stable ~2.5 years ago?

**Comment by trebmuh** [comment]
I've just checked, and it's clearly written [in the README here](https://github.com/olegkapitonov/Kapitonov-Plugins-Pack#dependencies-for-building) that you need a >2 version of Faust.

**Comment by tavasti** [comment]
Yeah, true, it says 2.x.

**Comment by olegkapitonov** [comment]
Yes, sorry but I use very new versions of packages and Arch Linux :)

**Comment by olegkapitonov** [comment]
Also, **zita-convolver** in Ubuntu 18.04 is old, 4.x required, so I created special branch for this - https://github.com/olegkapitonov/Kapitonov-Plugins-Pack/tree/thirdparty-included

This branch contains zita-convolver bundled. If you have meson-0.53 and faust 2.x built from source, you can build KPP from this branch.

**Comment by olegkapitonov** [comment]
With Ardour5 from Ubuntu repo all works on my setup.
Can you try this in Ardour?
I suspect that Mixbus 5 uses different, non-system version of GTK and this causes problem with zenity. Check in Ardour from repo is needed to proof this.
